src/WCSimAmBeGen.cc

```diff
@@ -0,0 +1,153 @@
+#include "WCSimAmBeGen.hh"
+#include "WCSimPrimaryGeneratorMessenger.hh"
+#include "WCSimPrimaryGeneratorAction.hh"
+
+#include "G4ParticleGun.hh"
+#include "G4PhysicalConstants.hh"
+#include "G4RandomDirection.hh"
+#include "G4SystemOfUnits.hh"
+#include "G4SPSEneDistribution.hh"
+#include "G4SPSRandomGenerator.hh"
+#include "G4ParticleTable.hh"
+#include "G4Neutron.hh"
+#include "G4Gamma.hh"
+
+#include <random>
+
+using namespace std;
+
+G4int    WCSimAmBeGen::nGammaOutcomes = 3;
+G4double WCSimAmBeGen::gammaProbabilities[3] = {0.26, 0.65, 0.08};
+G4double WCSimAmBeGen::gammaEnergies[3] = {0.0, 4.4, 7.7};
+G4int    WCSimAmBeGen::pdgids[2] = {2112, 22};
+
+WCSimAmBeGen::WCSimAmBeGen(){
+  // Initialise
+  this->Initialise();
+
+  wcsimdir = string(getenv("WCSIMDIR"))+"data/";
+
+  gs_path = wcsimdir + "ground_state_spectrum.txt";
+  fe_path = wcsimdir + "first_excited_spectrum.txt";
+  se_path = wcsimdir + "second_excited_spectrum.txt";
```

Comment:
No idea how you got so far when you were running it, when `gs_path` etc weren't even defined when you called `Initalise()`... for me it complained that it couldn't find the files

For me, this makes the setup work of these 3 things work (then it seg faults inside `GenerateNG()`... but some progress is good!) 
```suggestion
  wcsimdir = string(getenv("WCSIMDIR"))+"/data/";

  gs_path = wcsimdir + "ground_state_spectrum.txt";
  fe_path = wcsimdir + "first_excited_spectrum.txt";
  se_path = wcsimdir + "second_excited_spectrum.txt";
  
  // Initialise
  this->Initialise();
```

Reply:
Hi Tom, this changes allows me to run the simulation with no errors at all. And I even have the neutron energy spectrum I expected.
<img width="554" alt="Captura de pantalla 2024-03-04 a las 11 24 53" src="https://github.com/WCSim/WCSim/assets/67594492/a9cc25c6-caa5-4c57-9506-2bde85f43941">

Reply:
Great!